### PR TITLE
add: `truckersmp-cli`

### PR DESCRIPTION
**Pull request by badlydrawnface**

### CODE REVIEW

**Comment by badlydrawnface** [comment]
done

File: packages/truckersmp-cli/truckersmp-cli.pacscript [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name="truckersmp-cli"
+pkgname="truckersmp-cli"
+version=0.9.1.1
+repology=("project: truckersmp-cli")
+breaks=("truckersmp-cli")
+gives=("truckersmp-cli")
```
Comment (wizard-28) [inline packages/truckersmp-cli/truckersmp-cli.pacscript:6]:
```suggestion
gives="truckersmp-cli"
```

File: packages/truckersmp-cli/truckersmp-cli.pacscript [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name="truckersmp-cli"
+pkgname="truckersmp-cli"
+version=0.9.1.1
+repology=("project: truckersmp-cli")
+breaks=("truckersmp-cli")
```
Comment (wizard-28) [inline packages/truckersmp-cli/truckersmp-cli.pacscript:5]:
```suggestion
breaks="${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]
What about the `sdl2` dependency in the [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=truckersmp-cli)

**Comment by badlydrawnface** [comment]
added sdl dependancies, and a `break` variable only for a git version of this package, as this app doesn't come in binary format, neither as a deb or AppImage.

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**